api/externals/repository/fund_information_repository.go

```diff
@@ -49,7 +50,21 @@ func (fir *fundInformationRepository) Create(
 	isFirstCheck string,
 	isLastCheck string,
 ) error {
-	var query = "insert into fund_informations (user_id, teacher_id, price, remark, is_first_check, is_last_check) values ( " + userID + "," + teacherID + "," + price + ",'" + remark + "'," + isFirstCheck + "," + isLastCheck + ")"
+	query := `
+		INSERT INTO
+	 		fund_informations(
```

Comment:
```suggestion
			fund_informations(
```
変なスペースありました。これはgithub上で修正でよさそう